src/validations/admins.js

```diff
@@ -0,0 +1,83 @@
+const Joi = require('joi');
+
+const validateAdminCreate = (req, res, next) => {
```

Comment:
I think we need to be a little more consistent with the error messages in all validations, we can say that the name is invalid when is too long and later say that the last name is too long as a error for when the same thing happens, same for the validations below

Reply:
also we need to be constant if we are going to validate that we are using .base validation

Reply:
ok fede , I'm going to try to achieve that consistency in both strings. The comments about the error message were repeated. I use the same as another of the areas (trainers) since it was merged, to achieve that consistency that fails in my own code. Thank you for your marking, they are errors that I cannot have.